xbmc/input/ButtonTranslator.cpp

```diff
@@ -1170,6 +1170,12 @@ int CButtonTranslator::GetFallbackWindow(int windowID)
   return -1;
 }
 
+CAction CButtonTranslator::GetGlobalAction(const CKey &key) const
+{
+  std::string strAction;
+  return GetActionCode(-1, key, strAction);
```

Comment:
@ksooo please use `CButtonTranslator::GetActionCode(int window, int action)` to get rid of the dummy string var.